runtime/kusama/src/governance/mod.rs

```diff
@@ -14,7 +14,81 @@
 // You should have received a copy of the GNU General Public License
 // along with Polkadot. If not, see <http://www.gnu.org/licenses/>.
 
-//! Governance configurations for the Kusama runtime.
+//! New governance configurations for the Kusama runtime.
+
+use super::*;
+use frame_support::{
+	parameter_types,
+	traits::{ConstU16, EitherOf},
+};
+use frame_system::EnsureRootWithSuccess;
 
 // Old governance configurations.
 pub mod old;
+
+mod origins;
+pub use origins::{
+	pallet_custom_origins, AuctionAdmin, Fellows, FellowshipAdmin, FellowshipExperts,
+	FellowshipInitiates, FellowshipMasters, GeneralAdmin, LeaseAdmin, ReferendumCanceller,
+	ReferendumKiller, Spender, StakingAdmin, WhitelistedCaller,
+};
+mod tracks;
+pub use tracks::TracksInfo;
+mod fellowship;
+pub use fellowship::{FellowshipCollectiveInstance, FellowshipReferendaInstance};
+
+parameter_types! {
+	pub const VoteLockingPeriod: BlockNumber = 7 * DAYS;
+}
+
+impl pallet_conviction_voting::Config for Runtime {
+	type WeightInfo = pallet_conviction_voting::weights::SubstrateWeight<Self>; //TODO
+	type Event = Event;
+	type Currency = Balances;
+	type VoteLockingPeriod = VoteLockingPeriod;
+	type MaxVotes = ConstU32<512>;
+	type MaxTurnout = frame_support::traits::TotalIssuanceOf<Balances, Self::AccountId>;
+	type Polls = Referenda;
+}
+
+parameter_types! {
+	pub const AlarmInterval: BlockNumber = 1;
+	pub const SubmissionDeposit: Balance = 100 * UNITS;
+	pub const UndecidingTimeout: BlockNumber = 28 * DAYS;
+}
+
+parameter_types! {
+	pub const MaxBalance: Balance = Balance::max_value();
+}
+pub type TreasurySpender = EitherOf<EnsureRootWithSuccess<AccountId, MaxBalance>, Spender>;
+
+impl origins::pallet_custom_origins::Config for Runtime {}
+
+impl pallet_whitelist::Config for Runtime {
+	type WeightInfo = pallet_whitelist::weights::SubstrateWeight<Self>; //TODO
+	type Event = Event;
+	type Call = Call;
+	type WhitelistOrigin =
+		EitherOf<EnsureRootWithSuccess<Self::AccountId, ConstU16<65535>>, Fellows>;
```

Comment:
Does it mean that `Fellows` (from 3rd rank and higher) can let a `WhitelistedCaller` to execute any call as `Root`?
https://github.com/paritytech/substrate/blob/97ae6be11b0132224a05634c508417f048894670/frame/whitelist/src/lib.rs#L224

Reply:
No, it means that Root can call functions where we check for `WhistelistOrigin`: https://github.com/paritytech/substrate/blob/97ae6be11b0132224a05634c508417f048894670/frame/whitelist/src/lib.rs#L122